lib/FusionInventory/Agent/Task/Inventory/Win32/Bios.pm

```diff
@@ -53,6 +47,13 @@ sub doInventory {
         $bios->{BDATE}         = _dateFromIntString($object->{ReleaseDate});
     }
 
+    # Try to set Bios date from registry if not found via wmi
+    unless ($bios->{BDATE}) {
+        $bios->{BDATE} = _dateFromIntString(getRegistryValue(
+            path => "HKEY_LOCAL_MACHINE/Hardware/Description/System/BIOS/BIOSReleaseDate"
+        ));
+    }
+
```

Comment:
same comment: it seems indeed more efficient to only fallback on registry if needed, but that's still just a change in when to access the registry, not how it is accessed.

Reply:
I agree, comment needed here too.